src/events/models.py

```diff
@@ -0,0 +1,61 @@
+from django.db import models
+from utils.models import CreatedUpdatedModel
+from teams.models import Team
+
+class Type(CreatedUpdatedModel):
+    """
+    The events.Type model contains different types of system events which can happen.
+    New event types should be added in data migrations.
+    The following types are currently used in the codebase:
+    - ticket_created: Whenever a new ShopTicket is created
+    - public_credit_name_changed: Whenever a user changes public_credit_name in the profile
```

Comment:
What is the idea of having this in the database if we want to enforce that it requires a data migration to create an event?

Reply:
they have properties so it would get messy to use a choicefield

Reply:
After a discussion on IRC we decided to keep it like this :)